examples/perf-studio/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2016 - 2020 Sanity.io
```

Comment:
2021 :-)

Reply:
Copy/paste - but yeah. Let's instead update all licenses in a separate commit?

Reply:
Sure :-)